.github/scripts/get-refs-from-pr-body.js

```diff
@@ -0,0 +1,205 @@
+/**
```

Comment:
Is this the same script from https://github.com/smartcontractkit/chainlink-common/blob/main/.github/scripts/get-refs-from-pr-body.js ?

If so, would you be up for creating a small composite action on the /.github repo? I can help if needed.

Reply:
Created DX-1667 to track that 👍